deps/build.jl

```diff
@@ -21,20 +21,27 @@ isdir(TZDATA_DIR) || mkdir(TZDATA_DIR)
 isdir(COMPILED_DIR) || mkdir(COMPILED_DIR)
 
 info("Downloading TZ data")
-tzdata = ""
+archive = ""
 for url in URLS
     try
-        tzdata = download(url)
+        archive = download(url)
         break
     catch
         warn("Failed to download TZ data from: $url")
     end
 end
-isfile(tzdata) || error("Unable to download TZ data")
+isfile(archive) || error("Unable to download TZ data")
 
 info("Extracting TZ data")
-run(`tar xvf $tzdata -C $TZDATA_DIR $REGIONS`)
-rm(tzdata)
+@unix_only function extract(archive, directory, files)
+    run(`tar xvf $archive --directory=$directory $files`)
```

Comment:
As for the verbose option I think it's rather useful here as it shows which individual tzdata files we're extracting:
```
INFO: Extracting TZ data
x africa
x antarctica
x asia
x australasia
x europe
x northamerica
x southamerica
```